src/main/java/org/schemaspy/output/dot/schemaspy/DotConfigHeader.java

```diff
@@ -30,21 +30,21 @@
  * @author Rafal Kasa
  * @author Nils Petzaell
  */
-public final class DotFormat {
+public final class DotConfigHeader implements Header {
```

Comment:
I'm not sure that this is the correct direction.

dot-lang has some nesting not much but in our use-case it's one level. I'm no dot-lang expert so I don't know if digraph can be nested multiple levels, but it's not used by us at the moment.

So this might be better suited as `digraph` and it contains `nodes` and `edges`.

The main objection comes from the awkward code that looks like

```
dot.println(header.value());
dot.println(node.value());
dot.println("}");
```

The last line is iffy.

Reply:
I think you're assuming a direction that I didn't intend. I've renamed the class to better reflect what it represents, and introduced an abstraction.

> So this might be better suited as digraph and it contains nodes and edges.
>
> The main objection comes from the awkward code that looks like
>
> ```
> dot.println(header.value());
> dot.println(node.value());
> dot.println("}");
> ```
> The last line is iffy. 

I agree. The awkwardness was already there; this refactoring just made it more visible.